Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM node:8-alpine
-MAINTAINER butlerx <butlerx@notthe.cloud>
+MAINTAINER CoderDojo Foundation <webteam@coderdojo.org>
```

Comment:
Maintainer is deprecated in favor of label so it shoudl be `LABEL maintainer="CoderDojo Foundation <webteam@coderdojo.org>"